ab_tool/controllers.py

```diff
@@ -49,11 +53,32 @@ def intervention_point_url(request, intervention_point_id):
     return request.build_absolute_uri(reverse("ab_testing_tool_deploy_intervention_point",
                                               args=(intervention_point_id,)))
 
-def format_url(url):
+def validate_name(name):
+    if not name:
+        raise MISSING_NAME_PARAM
+    if len(name) > NAME_CHAR_LIMIT:
+        raise PARAM_LENGTH_EXCEEDS_LIMIT
+    return name
+
+
+def validate_weighting(weight):
+    if not 0 <= weight <=100:
+        raise INCORRECT_WEIGHTING_PARAM
+    return weight
+
+
+def validate_url(url):
```

Comment:
That's true for validate_url. I'll change validate_url to format_and_validate_url